spec/latest/provider-spec.yaml

```diff
@@ -0,0 +1,156 @@
+$schema: http://json-schema.org/draft-07/schema
```

Comment:
Similar as in case of super.json https://github.com/superfaceai/ast-js/blob/dev/src/interfaces/providerjson/providerjson.schema.json

Reply:
https://github.com/superfaceai/spec/pull/37#issuecomment-1060391474